backend/models/contact.model.js

```diff
@@ -0,0 +1,10 @@
+module.exports = (sequelize, Sequelize) => {
+    const Contact = sequelize.define("contacts", {
```

Comment:
I think we can change it to Contact, as sequelize has an in built pluralise feature which creates the table as contacts

Reply:
Noted on the above. 
From what I read in the documentation, sequelize `define` method calls `init` internally. And the auto pluralization only occurs when you don't use the `define` method and instead use the extend method and add the `init` on top of it. 

As such, would you prefer if we remove the `define` method and add the `init` instead?

**Reference documentation:** 
- [Sequelize define](https://sequelize.org/docs/v6/core-concepts/model-basics/#using-sequelizedefine)
- [Sequelize init](https://sequelize.org/docs/v6/core-concepts/model-basics/#extending-model)
- [Table name pluralization](https://sequelize.org/docs/v6/core-concepts/model-basics/#extending-model)

Reply:
Actually in this documentation:
You can stop the auto-pluralization performed by Sequelize using the freezeTableName: true option. This way, Sequelize will infer the table name to be equal to the model name, without any modifications:

sequelize.define('User', {
  // ... (attributes)
}, {
  freezeTableName: true
});

The example above will create a model named User pointing to a table also named User.
**So even if you do use define auto pluralise happens, it only stops if you use this attribute freezeTableName: true,** Alternatively you can do this as well:
sequelize.define('User', {
  // ... (attributes)
}, {
  tableName: 'Employees'
});

Reply:
Okay added `freezeTableName: true`
Context: since the table name was already pluralized, the sequelize doesn't pluralize it further. But in order to keep the model resilient and follow EE practices, it is best to freeze table name so auto pluralization doesn't override the preferred model name.